content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,163 @@
+---
+title: FinancialTransaction | R4 API
+---
+
+# FinancialTransaction
+
+* TOC
+{:toc}
+
+## Overview
+
+The FinancialTransaction resource provides the ability to record and exchange financial payment and adjustment information related to patient healthcare services.
+
+## Terminology Bindings
+
+<%= terminology_table(:financial_transaction, :r4) %>
+
+## Extensions
+
+* [Financial Transaction Account Number]
```

Comment:
Please make all the necessary links work for this table and the extension table below

Reply:
Are these supposed to link to the table below, since these fields don't exist under HL7

Reply:
Yes. Also the types should all be links to HL7 (the types in the table).

Reply:
Linked the types the HL7 field

Reply:
All of the links need to go back to R4

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR195

Reply:
+1